api/test/integration/test_cluster_endpoints.tavern.yaml

```diff
@@ -756,18 +756,16 @@ stages:
       json:
         error: !anyint
         data:
-          affected_items:
-            - name: worker1
-              status: 'OK'
-            - name: worker2
-              status: 'OK'
           failed_items:
             - error:
                 code: 1908
               id:
                 - 'master-node'
-          total_affected_items: 2
-          total_failed_items: 1
+                - 'worker1'
+                - 'worker2'
+          total_affected_items: 0
+          total_failed_items: 3
+    delay_before: 12
```

Comment:
Don't use `delay_before`. The delay must be set in the test case causing the possible race condition. In this case, the delay must be in the test `Upload corrupted configuration` and it should be a `delay_after`.

Add a variable to common.yaml as this value is repeated below.

Reply:
I would also suggest changing the delay value from 12 to 15 or 20 as it is only used twice and we could avoid future problems this way.